linux_os/guide/system/accounts/accounts-session/accounts_users_home_files_permissions/bash/shared.sh

```diff
@@ -0,0 +1,9 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+for home_dir in $(awk -F':' '{ if ($4 >= {{{ uid_min }}} && $4 != 65534) print $6 }' /etc/passwd); do
+    find $home_dir -perm /027 -exec chmod g-w,o=- {} \;
```

Comment:
Could we skip the `find` and just do `chmod -R g-w,o=-`?

Previous approach was doing the following, which could loosen permissions unintentionally.
` awk -F':' '{ if ($4 >= {{{ uid_min }}} && $4 != 65534) system("chmod -Rf 700 "$6) }' /etc/passwd`

But the following will just remove the offending permissions.
`awk -F':' '{ if ($4 >= {{{ uid_min }}} && $4 != 65534) system("chmod -Rf g-w,o=- "$6) }' /etc/passwd`